test/unit/rules/require-presentational-children-test.js

```diff
@@ -25,6 +25,10 @@ generateRuleTests({
       <title>Title here</title>
       <circle cx="10" cy="10" r="10"></circle>
     </svg>`,
+    {
+      config: { additionalNonSemanticTags: 'custom-element' },
```

Comment:
Isn't `additionalNonSemanticTags` supposed to be an array?

Reply:
Oops! I wrote this at 2am in my timezone, silly me. Will fix.